### PR TITLE
Convert Message.bodyBuffer from String to byte array

### PR DESCRIPTION
**Pull request by rkatsyuryna**

### CODE REVIEW

File: core/src/test/java/io/github/tcdl/msb/api/RequesterIT.java [hunk 1]
```diff
@@ -35,6 +36,17 @@ public void testRequestMessage() throws Exception {
         TestUtils.assertRequestMessagePayload(adapterJsonMessage, requestPayload, NAMESPACE);
     }
 
+    @Test()
```
Comment (vso-tc) [inline core/src/test/java/io/github/tcdl/msb/api/RequesterIT.java]: extra "()"

File: core/src/test/java/io/github/tcdl/msb/api/RequesterIT.java [hunk 2]
```diff
@@ -35,6 +36,17 @@ public void testRequestMessage() throws Exception {
...
+    public void testRequestMessageWithBodyBufferBase64Encoded() throws Exception {
+        Payload requestPayload = TestUtils.createSimpleRequestPayloadWithBodyBuffer(new byte[24]);
```
Comment (vso-tc) [inline core/src/test/java/io/github/tcdl/msb/api/RequesterIT.java]: I would create simple test that uses 1 or 2 bytes to make it more expressive

File: core/src/test/java/io/github/tcdl/msb/api/RequesterIT.java (hunk 1 above)
Comment (rkatsyuryna) [inline core/src/test/java/io/github/tcdl/msb/api/RequesterIT.java]: sure

File: core/src/test/java/io/github/tcdl/msb/api/RequesterIT.java (hunk 2 above)
Comment (rkatsyuryna) [inline core/src/test/java/io/github/tcdl/msb/api/RequesterIT.java]: tried to make same as in node.js version. but will update

**Pull request merged**